src/main/charts/bamboo-agent/Changelog.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## 1.5.0
+
+**Release date:** 2022-07-12
```

Comment:
the release date will be 2022-07-12 only if the release happens today, otherwise please update it and match with the release date.